lib/cm_admin/view_helpers/column_field_helper.rb

```diff
@@ -0,0 +1,29 @@
+module CmAdmin
+  module ViewHelpers
+    module ColumnFieldHelper
+
+      #adds prefix and suffix to a value
+      def add_prefix_and_suffix_helper(value, prefix, suffix)
+        prefix.to_s + ' ' + value.to_s + ' ' + suffix.to_s
```

Comment:
@charith08 - Can we use interpolation here? We don't have to convert every argument to a string and it would be more readable as well.
```suggestion
        "#{prefix} #{value} #{suffix}"
```